### ISSUE TITLE
Add jar file to artifacts

### ISSUE THREAD

**Issue by QuinnBast** [confluentinc/kafka-connect-jdbc#1122]
Having a zip/tgz of the source code files is nice, but it would be really nice if you provided a `.jar` file in the artifacts for direct download so that we don't have to compile the jar ourselves if we need it. This is specifically [for use with strimzi](https://strimzi.io/docs/operators/0.21.1/deploying.html#using-kafka-connect-with-plug-ins-str), but I'm sure there are other use cases as well.

**Comment by OneCricketeer** [comment]
You can use confluent-hub to install all the JARs needed for this connector. It's not a fat JAR, so it's not possible to download just one file

**Comment by QuinnBast** [comment]
I mean this jar:

> https://packages.confluent.io/maven/io/confluent/kafka-connect-jdbc/$KAFKA_JDBC_VERSION/kafka-connect-jdbc-$KAFKA_JDBC_VERSION.jar

Where `$KAFKA_JDBC_VERSION` is the version you want to download.

I had to find some super obscure [Dockerfile](https://github.com/debezium/debezium-examples/blob/master/unwrap-smt/debezium-jdbc-es/Dockerfile) from some [stack overflow question with only 1 view](https://stackoverflow.com/questions/62456098/how-to-add-kafka-connect-jdbc-5-5-0-jar-to-debezium-connect) which wasn't even related to my use case. It took me probably 1-2 hours to find this but using that URL format to download the jar I was able to get strimzi kafka connect working with jdbc. However, it would be nice if this jar, or even the URL, was available in a more visible location to others.
